site-pages/cart/cart.css

```diff
@@ -0,0 +1,113 @@
+.cart-container{
+    display: flex;
+    margin-top: 30px;
+}
+.cart-cards-container{
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 2.5rem;
+}
+
+.bill{
+    display: flex;
+    float: right;
+    width: 25%;
+    height: 100%;
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    padding: var(--content-padding);
+    margin-top: 30px;
+    margin-right: 20px;
+    justify-content: center;
+    align-items: center;
+}
+.bill-container{
+    display: flex;
+    flex-direction: column;
+
+}
+
+.cart-head{
+    font-size: 500;
+}
+.hori-img{
+    padding: 0.9375rem;
+    width: 15.625rem;
+    height: 16.875rem;
+}
+.discount{
+    text-decoration: line-through;
+    color: gray;
+}
+.discount-per{
+    color: var(--good-color);
+}
+.cart-card{
+    max-width: 90%;
+}
+.main-info-cont{
+    text-align: center;
+    align-items: center;
+    justify-content: center;
```

Comment:
Can make a common utility class named as dot-store-flex-center and place these properties inside it and use the same class in the HTML file

Reply:
Yep, making a single class for it.